contracts/axelar-gateway/tests/testdata/call_contract.golden

```diff
@@ -1,3 +1,9 @@
-contract: Contract(CAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAHK3M)
-topics: (Symbol(contract_called), Contract(CAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAITA4), String(ethereum), String(0x4EFE356BEDeCC817cb89B4E9b796dB8bC188DC59), BytesN<32>(86, 87, 13, 226, 135, 215, 60, 209, 203, 96, 146, 187, 143, 222, 230, 23, 57, 116, 149, 95, 222, 243, 69, 174, 87, 158, 233, 244, 117, 234, 116, 50))
-data: (Bytes(18, 52))
\ No newline at end of file
+Contract(CAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAHK3M)
+
+ContractCalledEvent {
+    caller: Contract(CAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAITA4),
+    destination_chain: String(ethereum),
+    destination_address: String(0x4EFE356BEDeCC817cb89B4E9b796dB8bC188DC59),
+    payload_hash: BytesN<32>(86, 87, 13, 226, 135, 215, 60, 209, 203, 96, 146, 187, 143, 222, 230, 23, 57, 116, 149, 95, 222, 243, 69, 174, 87, 158, 233, 244, 117, 234, 116, 50),
+    payload: Bytes(18, 52),
```

Comment:
Looking into tagging data fields with `#[data]` to catch changes to what goes in data vs topics. Ordering changes should be visible based on the struct fields

Reply:
If tagging `#[data]` is trickier, another option is to print the tuple of types that goes into `topics` and `data`